src/styles/CardStyle/_card.scss

```
@@ -0,0 +1,127 @@
+@use './variables';
+
+/* #region Card Component */
+.card {
+  width: 198px;
```

you're using box-sizing: border-box so width includes border 
```suggestion
  width: 200px;
```